README.md

```diff
@@ -0,0 +1,70 @@
+# sentayhu19-Portfolio-setup-and-mobile-version-skeleton
+![](https://img.shields.io/badge/Microverse-blueviolet)
+# Portofolio
+ 
+> Desgining a portofolio 
+
+## Built With
+
+-Lighthouse (An open-source, automated tool for improving the quality of web pages. It has audits for performance, accessibility, progressive web apps, SEO and more).
+-Webhint (A customizable linting tool that helps you improve your site's accessibility, speed, cross-browser compatibility, and more by checking your code for best practices and common errors).
+-Stylelint (A mighty, modern linter that helps you avoid errors and enforce conventions in your styles).
+
+
+# Getting Started
+
+#### Cloning the project
+
+git clone  https://github.com/sentayhu19/portofolio-setup-nd-mobile-version-skeleton <Your-Build-Directory>
+
+
+## Getting packages and debuging with Stylelint
+
+npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+
+##### For validation detection using Stylelint Run
+
+npx stylelint "**/*.{css,scss}"
+
+##### from parent source directory
+
+## Getting packages and debuging with Webhint
+
+npm init -y
+npm install --save-dev hint@6.x
+
+##### For validation detection using Webhint Run
+
+npx hint . 
+
+### Prerequisites
+##VSCode or any other equivalent code editor
+
+##node package manager
+### Licence
+MIT License
+##### from parent source directory
+Copyright (c) 2022 Sentayhu
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+
+
+
+## Author
+ **Sentayhu Berhanu Robi** - [sentayhhu19](https://github.com/sentayhu19)
```

Comment:
- [ ] Please make sure you stick to the [Readme template](https://github.com/microverseinc/readme-template/blob/master/README.md) provided by Microverse for a professional documentation, you can edit your current Readme and fill it with the missing sections.